app/models/log_count.rb

```diff
@@ -0,0 +1,5 @@
+class LogCount < ActiveRecord::Base
+    scope: two_months_ago do
+        where(created_at: 2.months.ago)
+    end
+end
```

Comment:
Do we need an entire new model to restrict the amount of logs shown? Could we add a scope to the existing Log model; then in the part of the code that makes the histogram for a project call `project.logs.up_to_two_months_ago`